app/views/insured/families/_enrollment.html.erb

```diff
@@ -155,7 +155,7 @@
       <div>
       </div>
       <div class="enrollment-type info vertical-align-top">
-        <% if osse_eligibility_is_enabled? %>
+        <% if ivl_osse_eligibility_is_enabled? || shop_osse_eligibility_is_enabled? %>
```

Comment:
@jacobkagon We will probably have to check for the hbx_enrollment kind (employer_sponsored vs individual) and based on that we can check for osse flag to see if its enabled.